Meta/write-only-on-difference.sh

```diff
@@ -1,28 +0,0 @@
-#!/bin/sh
```

Comment:
/me waves goodbye

It's good to see that it is being replaced by a proper pointless-rebuild-avoidance-mechanism. :)